services/datamanager/builtin/capture_test.go

```diff
@@ -57,14 +56,7 @@ func TestDataCaptureEnabled(t *testing.T) {
 			initialCollectorDisableStatus: false,
 			newCollectorDisableStatus:     false,
 		},
-		{
-			name:                          "data capture service implicitly enabled and a configured collector, should capture data",
```

Comment:
As Aaron would say, yeeted 💥 

Could we replace this with a test that without a data capture service, we do not capture data?

Reply:
I added a replacement test that checks to see if an empty config is made, no data manager is made which implicitly satisfies that no data is being captured. Kind of hard to check data being captured with no data manager.

Reply:
It's called TestEmptyConfig or smth